llama_stack/providers/remote/vector_io/qdrant/qdrant.py

```diff
@@ -263,3 +268,38 @@ async def openai_attach_file_to_vector_store(
         chunking_strategy: VectorStoreChunkingStrategy | None = None,
     ) -> VectorStoreFileObject:
         raise NotImplementedError("OpenAI Vector Stores API is not supported in Qdrant")
+
+    async def openai_list_files_in_vector_store(
+        self,
+        vector_store_id: str,
+    ) -> VectorStoreListFilesResponse:
+        raise NotImplementedError("OpenAI Vector Stores API is not supported in Qdrant")
```

Comment:
Think this causes a failure during llama stack build (was true I believe for Milvus as well and I removed in my PR   https://github.com/meta-llama/llama-stack/pull/2470

Reply:
Hmm - `llama stack build` calls `openai_list_files_in_vector_store`? I haven't run into this myself - but also perhaps have not built a distribution that uses Qdrant. How would I trigger the error you hit in Milvus?